### PR TITLE
[16.0] [MIG] server_environment_ir_config_parameter

### PR DESCRIPTION
**Pull request by baimont**
https://github.com/OCA/server-env/issues/120

depends on https://github.com/OCA/server-env/pull/122

### CODE REVIEW

File: server_environment_ir_config_parameter/tests/config_param_test.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<odoo>
+    <record model="ir.config_parameter" id="some_record_id">
```
Comment (acsonefho) [inline server_environment_ir_config_parameter/tests/config_param_test.xml:2]: Should be something else, no? `some_record_id`

**Review by acsonefho — CHANGES_REQUESTED** [review]

File: server_environment_ir_config_parameter/tests/config_param_test.xml (hunk 1 above)
Comment (baimont) [reply server_environment_ir_config_parameter/tests/config_param_test.xml:2]: it's just for tests - the xml_id is not important. what matters is the key and value

**Comment by florian-dacosta** [comment]
Hello @baimont 
I think this PR is missing and should be applied : https://github.com/OCA/server-env/pull/92/files
It was merged in 14.0 after the migration of 15.0 and therefore is missing.

**Comment by baimont** [comment]
> Hello @baimont I think this PR is missing and should be applied : https://github.com/OCA/server-env/pull/92/files It was merged in 14.0 after the migration of 15.0 and therefore is missing.

Hello @florian-dacosta, I cherry-picked the commit you refer to. Thanks for your review

**Review by rven — APPROVED** [review]

**Review by lmignon — APPROVED** [review]
Thank you @baimont :+1: (Code review)

Waiting for the merge of #122

**Comment by sebalix** [comment]
Commit acca2a2a9f061b7e68840d2ee85ecd78210c4d20 to remove and we are good.

**Comment by baimont** [comment]
> Commit [acca2a2](https://github.com/OCA/server-env/commit/acca2a2a9f061b7e68840d2ee85ecd78210c4d20) to remove and we are good.

done

**Review by sebalix — APPROVED** [review]

**Pull request merged**